data/data/gcp/bootstrap/main.tf

```diff
@@ -61,6 +68,16 @@ data "ignition_config" "redirect" {
   }
 }
 
+resource "google_tags_location_tag_binding" "user_tag_binding_bucket" {
+  for_each = var.gcp_extra_tags
+
+  parent = format("//storage.googleapis.com/projects/_/buckets/%s",
+    google_storage_bucket.ignition.name,
+  )
```

Comment:
It seems strange that there is an underscore here in place for the project. Can you provide any more context for that?

Reply:
GCP Buckets aren't project scoped but is global resource, hence I think API is not expecting the project ID. More details on the bucket scope can be found [here](https://cloud.google.com/storage/docs/json_api/v1/buckets) and details on API for adding tags to buckets can be found [here](https://cloud.google.com/storage/docs/tags-and-labels#tags)